static/app/views/issueDetails/allEventsTable.tsx

```diff
@@ -164,6 +164,11 @@ const getPlatformColumns = (
   const platformCategory = platformToCategory(platform);
   const platformColumns = categoryToColumnMap[platformCategory];
 
+  if (options.isReplayEnabled) {
+    platformColumns.fields.push(...replayField);
+    platformColumns.columnTitles.push(...replayColumnTitle);
+  }
```

Comment:
the earlier lines at the top of the fund that read `options.isReplayEnabled` could probably be merged down into here.

Reply:
haha, oh man i must've been on autopilot. i'll clean it up